hw4/01-routing.js

```diff
@@ -31,18 +31,57 @@ let getRoutes = () => {
   return result;
 };
 
-app.get('/', (req, res) => {
+app.get("/", (req, res) => {
   let routeResults = getRoutes();
 
-  res.writeHead(200, { 'Content-Type': 'text/html' });
+  res.writeHead(200, { "Content-Type": "text/html" });
   res.write(`<h1>Exercise 04</h1>`);
   res.write(`<ul> ${routeResults} </ul>`);
   res.end();
 });
 
```

Comment:
I did the same thing but we have res.status and res.set in express, we have to use that instead of plain node js